### PR TITLE
chore: add Aspect Workflows

### PR DESCRIPTION
**Pull request by gregmagolan**
Onboard rules_terser to Aspect Workflows on GCP + CircleCI.

### CODE REVIEW

File: WORKSPACE.bazel [hunk 1]
```diff
@@ -0,0 +1,5 @@
+############################################
```
Comment (jbedard) [inline WORKSPACE.bazel:1]: Isn't this repo bzlmod-only now?

File: .aspect/cli/config.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+configure:
+  languages:
+    javascript: true
```
Comment (jbedard) [inline .aspect/cli/config.yaml:3]: You're sure we want this on?

**Review by jbedard — APPROVED** [review]

File: .aspect/cli/config.yaml (hunk 2 above)
Comment (gregmagolan) [reply .aspect/cli/config.yaml:3]: Yes. I need to coverage for the `configure` task which depends on `aspect configure` working even if not generating anything yet.

File: WORKSPACE.bazel (hunk 1 above)
Comment (gregmagolan) [reply WORKSPACE.bazel:1]: Still need one non-bzlmod dep for rules_workflows. Will publish that to BCR in the future.

**Comment by kormide** [comment]
Is there a reason to not use `rules_workflows` for generating the circle config? Is it just because 5.11 isn't supported yet?

**Comment by gregmagolan** [comment]
> Is there a reason to not use `rules_workflows` for generating the circle config? Is it just because 5.11 isn't supported yet?

I'm planning to upgrade these to use rules_workflows soon in a follow-up.

**Comment by gregmagolan** [comment]
> > Is there a reason to not use `rules_workflows` for generating the circle config? Is it just because 5.11 isn't supported yet?
> 
> I'm planning to upgrade these to use rules_workflows soon in a follow-up. rules_workflows API needed finalization which was is why I hadn't rolled it out broadly yet.

**Pull request merged**